Bug fix + unduplicate code

daniel-wells opened the pull request:
Hi Andrea, thanks for the great package!

I had a couple of issues when I tried to use it that I've fixed in this PR:
- If the path to pem file had spaces in it e.g. "/Users/wells/Dropbox (Personal)/Github/shinyexample/danielwells.pem" an error was thrown "unexpected (", so I've wrapped file paths with additional single quotes to prevent this.
- The update process was hanging on the st andrews cran server as it wasn't responding in time, I've changed this to the Rstudio cran server.

I also made another couple of changes which are not fixing bugs but generally tidying the code:
- Some code was duplicated across the ramazon and ramazon_update functions so I've moved this into seperate functions so they only have to be updated/maintained in one place
- I added automatic detection of the .pem file if it's in the current working directory with the other app files
- I tried to use paste0 consistently rather than paste(sep="") in some places and paste0() elsewhere

daniel-wells committed:
remove unused .RData files

daniel-wells committed:
use Rstudio cran server

st-andrews was taking 10-20 mins to respond

daniel-wells committed:
use paste0 consistently

rather than paste(sep="") sometimes and paste0 others

daniel-wells committed:
put duplicated code into functions & automatically detect key pair file

Code was duplicated over functions ramazon and ramazon_update, so put
them in their own functions. Automatically detect key pair file in
current directory if not specified

AndreaCirilloAC reviewed (changes requested):
@daniel-wells thank you for this commit. I would not go for the super assignment since it will make the 'current' and the key_pair_name objects last even after the end of ramazon running.
I would rather place a return(list(current,key_pair_name)) at the end of the setup_connection_variables() function.
Let me know if this is clear enough

AndreaCirilloAC reviewed (changes requested):
@daniel-wells thank you for this commit. I would not go for the super assignment since it will make the 'current' and the key_pair_name objects last even after the end of ramazon running.
I would rather place a return(list(current,key_pair_name)) at the end of the setup_connection_variables() function.
Let me know if this is clear enough

daniel-wells committed:
remove super assignment

Re-organised function setup_connection_variables to return a single
variable: key_pair_address.

The user_server variable is only required in the copy_files_to_server
function so can delay creation until then.

'current' is also not used outside the setup_connection_variables
function.

So only one variable left to return, so super assignment redundant and
can rename function to get_key_pair_address.

daniel-wells commented:
Thanks for the suggestion I agree and it actually made me realise super assignment was redundant anyway!

AndreaCirilloAC commented:
well done @daniel-wells, thank you.
